mono/utils/mono-coop-mutex.h

```diff
@@ -109,13 +109,27 @@ mono_coop_cond_timedwait (MonoCoopCond *cond, MonoCoopMutex *mutex, guint32 time
 static inline void
 mono_coop_cond_signal (MonoCoopCond *cond)
 {
+	/*
+	 * On glibc using NTPL (ie Linux with an underlying futex), signaling a
+	 * condition variable can block in the __condvar_quiesce_and_switch_g1
+	 * operation.  So switch to GC Safe mode here.
```

Comment:
Modern style is one space after periods.

Reply:
And this seems reasonable even with less commentary -- "syscalls can block" -- klnda just a blanket/blind good assumption. But ok.

Reply:
While it isn't likely, it could be they haven't been called in a while, are paged out, and just calling them requires paging them in. Just crossing from one page of code to another, without even a call, can page fault and be slow. But we can't do much about that, except maybe page-align functions and keep them under page in size, limiting the fault to their start.